src/views/onboardingProduct/OnboardingProduct.tsx

```diff
@@ -195,6 +197,20 @@ function OnboardingProductComponent({ productId }: { productId: string }) {
     }
   }, [onboardingFormData]);
 
+  useEffect(() => {
```

Comment:
It would be better to act on the api call:
- Invocation when we have institutionType and productId data
- Filter the response getting what we need
- Pass the state variable to the components used